docs/src/pages/en/styling-reference.astro

```diff
@@ -0,0 +1,107 @@
+---
+import { Markdown } from '@astropub/md';
+import MainLayout from '../../layouts/MainLayout.astro';
+import manifest from '../../../../dist/custom-elements.json';
+
+const { frontmatter = {}, headings = [], url } = Astro.props as Props;
+const components = await Astro.glob('./components/*');
+
+function hasLink(tagName) {
+  return components.some(c => c.file.includes(tagName));
+}
+
+const groups = [
+  'button',
+  'range',
+  'display',
+  'indicator',
+  'other',
```

Comment:
I think this bucket will confuse folks, since not every CSS variable will apply to every "other" component. May want to treat this group differently with how you render, even if it's just a clear up-front caveat in text?

Reply:
good call, will change this